lib/utils/getOverlayEntry.js

```diff
@@ -0,0 +1,57 @@
+const querystring = require('querystring');
+
+/**
+ * Creates an object that contains two entry arrays: the prependEntries and overlayEntries
+ * @param {import('../types').NormalizedPluginOptions} options Configuration options for this plugin.
+ * @param {import('webpack').Compiler.options.devServer | undefined} devServer The webpack devServer config
```

Comment:
JSDoc specific syntax

```suggestion
 * @param {import('webpack').Compiler.options.devServer} [devServer] The webpack devServer config
```